### PR TITLE
output schema fixes

### PR DESCRIPTION
**Pull request by karenetheridge**
Fixed some errors in output/schema.json

### CODE REVIEW

**Comment by gregsdennis** [comment]
@karenetheridge Could you link to the portions of the spec that this file references, please?  Just to make sure that this change is valid.

**Comment by karenetheridge** [comment]
re absoluteKeywordLocation: https://json-schema.org/draft/2019-09/json-schema-core.html#rfc.section.10.3.2

re patterns: https://json-schema.org/draft/2019-09/json-schema-core.html#rfc.section.6.4

**Comment by gregsdennis** [comment]
@karenetheridge thanks, but I mean the output portions.  Where are the schema and instance that generated the output (they're in the spec, but I'm too busy to find them right now)?

File: output/schema.json [hunk 1]
```diff
@@ -46,18 +46,18 @@
         },
         {
           "if": {
-            "oneOf": [
+            "anyOf": [
```
Comment (gregsdennis) [inline output/schema.json:49]: Why this change?  Are you expecting `$ref` and `$recursiveRef` to both appear in the keyword location?

... I suppose they could....

okay.

Comment (karenetheridge) [reply output/schema.json:49]: sure, if there is a $recursiveRef in the subschema at the other end of a $ref, it could certainly happen.

I have a test case with `.../$ref/$ref/$ref/$ref/..` :) -- https://github.com/karenetheridge/JSON-Schema-Draft201909/blob/master/t/errors.t#L788

**Comment by Relequestual** [comment]
@gregsdennis If you'll approve, let's merge this?
